tests/Unit/DataStructures/Test_DataVector.cpp

```diff
@@ -28,9 +27,9 @@
 #endif
 }
 
-// [[OutputRegex, Must copy into same size]]
-[[noreturn]] SPECTRE_TEST_CASE("Unit.DataStructures.DataVector.RefDiffSize",
-                               "[DataStructures][Unit]") {
+    // [[OutputRegex, Must copy into same size]]
```

Comment:
Whoops, clang-format plz :(

Reply:
ugh, clang format breaks those for me every time... fixing...